charts/karpenter/templates/deployment.yaml

```
@@ -76,10 +76,18 @@ spec:
               value: "1.19.0-0"
             - name: KARPENTER_SERVICE
               value: {{ include "karpenter.fullname" . }}
+          {{- if .Values.webhook.enabled }}
             - name: WEBHOOK_PORT
               value: "{{ .Values.webhook.port }}"
             - name: WEBHOOK_METRICS_PORT
               value: "{{ .Values.webhook.metrics.port }}"
+            - name: DISABLE_WEBHOOK
+              value: "false"
```

```suggestion
              value: {{ not .Values.webhook.enabled | quote }}
```

Consider simplifying this here